### PR TITLE
PFM-2501:: COLLECTION_OPERATOR role added to Create and Update Consumers

### PR DESCRIPTION
**Pull request by Ramkrishna-egov**
[PFM-2501](https://digit-discuss.atlassian.net/browse/PFM-2501)

[PFM-2501]: https://digit-discuss.atlassian.net/browse/PFM-2501?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: frontend/mgramseva/lib/utils/role_actions.dart [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import 'dart:math';
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/utils/role_actions.dart:1]: why import removed is it un-used. ?

File: frontend/mgramseva/lib/utils/role_actions.dart [hunk 2]
```diff
@@ -123,10 +120,6 @@ class RoleActionsFiltering {
         return ['EXPENSE_PROCESSING', 'SUPERUSER'];
       case Routes.EXPENSE_UPDATE:
         return ['EXPENSE_PROCESSING', 'SUPERUSER'];
-
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/utils/role_actions.dart:126]: can you attach the ticket link for this ?

**Review by naveen-egov — CHANGES_REQUESTED** [review]

File: frontend/mgramseva/lib/utils/role_actions.dart (hunk 1 above)
Comment (Ramkrishna-egov) [reply frontend/mgramseva/lib/utils/role_actions.dart:1]: Yes its unused, Latest version of Android studio removes unused imports on commit,

File: frontend/mgramseva/lib/utils/role_actions.dart (hunk 2 above)
Comment (Ramkrishna-egov) [reply frontend/mgramseva/lib/utils/role_actions.dart:126]: Sure

**Review by naveen-egov — APPROVED** [review]

**Pull request merged**